### PR TITLE
Add conditional lighting on SplashPage

### PR DESCRIPTION
**Pull request by mariomui**
When you move across each image, the other images darken while the
current Picture brightens.

### CODE REVIEW

File: client/components/CarouselCard.jsx [hunk 1]
```diff
@@ -11,6 +11,13 @@ const Image = styled.div`
   min-height: 100%;
   width: 500px;
   height: 300px;
+  ${({ currPicIdx, avalue }) => (currPicIdx !== avalue) && css`
```
Comment (johnsonkwok) [inline client/components/CarouselCard.jsx]: Perhaps you can rename "avalue" to "aValue" for clarity. Wasn't sure if that was supposed to be one word

File: client/components/CarouselCard.jsx [hunk 2]
```diff
@@ -43,10 +50,20 @@ const OverFlowWrap = styled.div`
...
 
 const CarouselCard = (props) => {
-  const { inModal, imageUrl, avalue, handleModalClick } = props;
+  const { currPicIdx, handleHover, inModal, imageUrl, avalue, handleModalClick } = props;
   return (
     <OverFlowWrap inModal={inModal}>
-      <Image inModal={inModal} onClick={(e) => { handleModalClick(e, avalue); }} avalue={avalue} imageUrl={imageUrl} key={avalue} alt="pictures" />
+      <Image
+        currPicIdx={currPicIdx}
+        onFocus={(e) => { handleHover(e, avalue); }}
```
Comment (johnsonkwok) [inline client/components/CarouselCard.jsx]: Same comment as above regarding "avalue", some uncertainty as to what that is

File: client/components/CarouselLanding.jsx [hunk 3]
```diff
@@ -57,19 +60,40 @@ class CarouselLanding extends React.Component {
       .then(response => (response.data));
   }
 
+  handleHover(e, currPicIdx) {
+    this.setState({ currPicIdx: currPicIdx });
```
Comment (johnsonkwok) [inline client/components/CarouselLanding.jsx]: I believe you can also write this line as:
`this.setState({ currPicIdx });`
to make it more concise

Comment (mariomui) [reply client/components/CarouselLanding.jsx]: Roger That.

**Pull request merged**